src/data/pages/exploring-lsst-data-1.json

```diff
@@ -23,6 +23,7 @@
       },
       "options": {
         "title": "Color Vs Distance",
+        "showFlux": true,
```

Comment:
This is a sensible strategy to get this boolean where you need it, but I really think this is a bit of a slippery slope.  We've done a good job of abstracting the BrightnessVsDistance plot to be a GalacticPropertiesPlot but I think it's still better to rely on the x and y valueAccessors and let those inform point styling.  So you could even still use this `showFlux` flag internal to the components if you want, but I feel like `showFlux` is a little out of place in the pageData.  Instead let's just set the flag to true is the yValueAccessor is 'color'.

Reply:
Okay, removed showFlux from the data and all places it 'touched' on it's way into the Points.jsx

All should be good. I've restarted and cleaned the gatsby to check for error with none found.